content/resources/beginners.md

```diff
@@ -39,10 +39,36 @@ library("swirl")
   + [R for Social Scientists](https://datacarpentry.org/r-socialsci/)
   + [R for Ecologists](https://datacarpentry.org/R-ecology-lesson/)
 
+## Presentations/videos
```

Comment:
All of these are great, but I would consider most of them to be more advanced topics (and the presentation might be hard to follow).